[2022-06-25 00:05:47,200: INFO: stage_02_featurization]: 
********************
[2022-06-25 00:05:47,201: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-25 00:05:47,203: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 00:05:47,204: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 00:05:47,204: INFO: common]: created directory at: artifacts\features
[2022-06-25 00:05:47,432: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-25 00:05:47,432: ERROR: stage_02_featurization]: 'DataFrame' object has no attribute 'text'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 50, in main
    train_words=np.array(df_train.text.str.lower().values.astype('U'))
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5141, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'text'
[2022-06-25 00:07:35,930: INFO: stage_02_featurization]: 
********************
[2022-06-25 00:07:35,931: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-25 00:07:35,932: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 00:07:35,933: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 00:07:35,933: INFO: common]: created directory at: artifacts\features
[2022-06-25 00:07:36,158: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-25 00:07:36,158: ERROR: stage_02_featurization]: 'DataFrame' object has no attribute 'text'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 50, in main
    train_words=np.array(df_train.text.str.lower().values.astype('U'))
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5141, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'text'
[2022-06-25 14:01:37,662: INFO: stage_02_featurization]: 
********************
[2022-06-25 14:01:37,664: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-25 14:01:37,672: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 14:01:37,673: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 14:01:37,675: INFO: common]: created directory at: artifacts\features
[2022-06-25 14:01:37,985: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-25 14:01:37,988: ERROR: stage_02_featurization]: 'DataFrame' object has no attribute 'text'
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 50, in main
    train_words=np.array(df_train.text.str.lower().values.astype('U'))
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5141, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'text'
[2022-06-25 14:06:39,659: INFO: stage_01_prepare_data]: 
********************
[2022-06-25 14:06:39,660: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-25 14:06:39,662: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 14:06:39,663: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 14:06:39,664: INFO: common]: created directory at: artifacts\prepared
[2022-06-25 14:06:42,373: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-25 14:15:20,774: INFO: stage_01_prepare_data]: 
********************
[2022-06-25 14:15:20,775: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-25 14:15:20,777: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 14:15:20,779: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 14:15:20,779: INFO: common]: created directory at: artifacts\prepared
[2022-06-25 14:15:23,960: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-25 14:16:51,795: INFO: stage_01_prepare_data]: 
********************
[2022-06-25 14:16:51,795: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-25 14:16:51,798: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-25 14:16:51,799: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-25 14:16:51,799: INFO: common]: created directory at: artifacts\prepared
[2022-06-25 14:16:55,000: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-26 15:40:45,410: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:40:45,411: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:40:45,414: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:40:45,416: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:40:45,416: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:40:45,631: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:41:01,007: ERROR: stage_02_featurization]: Cannot convert non-finite values (NA or inf) to integer
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 65, in main
    save_matrix(df=df_train, text_matrix= train_words_tfidf_matrix , out_path=featurized_train_data_path)
  File "c:\users\lmohan2\dvc-project-docs-template\lm_dvc_nlp_project-with-docs\src\utils\data_mgmt.py", line 36, in save_matrix
    pid_matrix=sparse.csr_matrix(df.pid.astype(np.int64)).T
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5548, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors,)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 604, in astype
    return self.apply("astype", dtype=dtype, copy=copy, errors=errors)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 409, in apply
    applied = getattr(b, f)(**kwargs)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\blocks.py", line 595, in astype
    values = astype_nansafe(vals1d, dtype, copy=True)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\dtypes\cast.py", line 968, in astype_nansafe
    raise ValueError("Cannot convert non-finite values (NA or inf) to integer")
ValueError: Cannot convert non-finite values (NA or inf) to integer
[2022-06-26 15:42:56,698: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:42:56,698: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:42:56,700: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:42:56,701: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:42:56,701: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:42:56,918: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:44:29,978: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:44:29,978: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:44:29,980: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:44:29,981: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:44:29,982: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:44:30,196: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:44:44,639: ERROR: stage_02_featurization]: Cannot convert non-finite values (NA or inf) to integer
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 65, in main
    save_matrix(df=df_train, text_matrix= train_words_tfidf_matrix , out_path=featurized_train_data_path)
  File "c:\users\lmohan2\dvc-project-docs-template\lm_dvc_nlp_project-with-docs\src\utils\data_mgmt.py", line 36, in save_matrix
    pid_matrix=sparse.csr_matrix(df.pid.astype(np.int64)).T
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5548, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors,)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 604, in astype
    return self.apply("astype", dtype=dtype, copy=copy, errors=errors)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 409, in apply
    applied = getattr(b, f)(**kwargs)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\blocks.py", line 595, in astype
    values = astype_nansafe(vals1d, dtype, copy=True)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\dtypes\cast.py", line 968, in astype_nansafe
    raise ValueError("Cannot convert non-finite values (NA or inf) to integer")
ValueError: Cannot convert non-finite values (NA or inf) to integer
[2022-06-26 15:47:08,069: INFO: stage_01_prepare_data]: 
********************
[2022-06-26 15:47:08,069: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-26 15:47:08,072: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:47:08,076: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:47:08,077: INFO: common]: created directory at: artifacts\prepared
[2022-06-26 15:47:11,647: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-26 15:48:07,974: INFO: stage_01_prepare_data]: 
********************
[2022-06-26 15:48:07,974: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-26 15:48:07,977: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:48:07,978: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:48:07,978: INFO: common]: created directory at: artifacts\prepared
[2022-06-26 15:48:11,507: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-26 15:48:14,523: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:48:14,523: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:48:14,526: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:48:14,527: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:48:14,529: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:48:14,772: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:48:29,332: ERROR: stage_02_featurization]: Cannot convert non-finite values (NA or inf) to integer
Traceback (most recent call last):
  File "src/stage_02_featurization.py", line 87, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_02_featurization.py", line 65, in main
    save_matrix(df=df_train, text_matrix= train_words_tfidf_matrix , out_path=featurized_train_data_path)
  File "c:\users\lmohan2\dvc-project-docs-template\lm_dvc_nlp_project-with-docs\src\utils\data_mgmt.py", line 36, in save_matrix
    pid_matrix=sparse.csr_matrix(df.pid.astype(np.int64)).T
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\generic.py", line 5548, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors,)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 604, in astype
    return self.apply("astype", dtype=dtype, copy=copy, errors=errors)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\managers.py", line 409, in apply
    applied = getattr(b, f)(**kwargs)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\internals\blocks.py", line 595, in astype
    values = astype_nansafe(vals1d, dtype, copy=True)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\pandas\core\dtypes\cast.py", line 968, in astype_nansafe
    raise ValueError("Cannot convert non-finite values (NA or inf) to integer")
ValueError: Cannot convert non-finite values (NA or inf) to integer
[2022-06-26 15:52:22,164: INFO: stage_01_prepare_data]: 
********************
[2022-06-26 15:52:22,164: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data started <<<<<
[2022-06-26 15:52:22,166: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:52:22,167: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:52:22,168: INFO: common]: created directory at: artifacts\prepared
[2022-06-26 15:52:25,582: INFO: stage_01_prepare_data]: >>>>> stage Prepare_data completed!<<<<<

[2022-06-26 15:52:28,215: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:52:28,215: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:52:28,218: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:52:28,219: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:52:28,220: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:52:28,472: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:52:43,835: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\train.pkl of shape:(17450, 2502)
[2022-06-26 15:52:43,982: INFO: common]: The input data frame artifacts\prepared\test.tsv of size (7550, 3) is read.
[2022-06-26 15:52:47,287: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\test.pkl of shape:(7550, 2502)
[2022-06-26 15:52:47,581: INFO: stage_02_featurization]: >>>>> stage Featurization completed!<<<<<

[2022-06-26 15:55:03,908: INFO: stage_02_featurization]: 
********************
[2022-06-26 15:55:03,908: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 15:55:03,911: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 15:55:03,912: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 15:55:03,912: INFO: common]: created directory at: artifacts\features
[2022-06-26 15:55:04,151: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 15:55:18,921: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\train.pkl of shape:(17450, 2502)
[2022-06-26 15:55:19,039: INFO: common]: The input data frame artifacts\prepared\test.tsv of size (7550, 3) is read.
[2022-06-26 15:55:21,812: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\test.pkl of shape:(7550, 2502)
[2022-06-26 15:55:22,142: INFO: stage_02_featurization]: >>>>> stage Featurization completed!<<<<<

[2022-06-26 20:21:55,430: INFO: stage_02_featurization]: 
********************
[2022-06-26 20:21:55,430: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 20:21:55,433: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 20:21:55,435: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 20:21:55,435: INFO: common]: created directory at: artifacts\features
[2022-06-26 20:21:55,666: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 20:22:08,926: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\train.pkl of shape:(17450, 2502)
[2022-06-26 20:22:09,030: INFO: common]: The input data frame artifacts\prepared\test.tsv of size (7550, 3) is read.
[2022-06-26 20:22:11,267: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\test.pkl of shape:(7550, 2502)
[2022-06-26 20:22:11,552: INFO: stage_02_featurization]: >>>>> stage Featurization completed!<<<<<

[2022-06-26 20:28:43,953: INFO: stage_02_featurization]: 
********************
[2022-06-26 20:28:43,955: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 20:28:43,957: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 20:28:43,958: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 20:28:43,959: INFO: common]: created directory at: artifacts\features
[2022-06-26 20:28:44,176: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 20:28:57,819: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\train.pkl of shape:(17450, 2502)
[2022-06-26 20:28:57,929: INFO: common]: The input data frame artifacts\prepared\test.tsv of size (7550, 3) is read.
[2022-06-26 20:29:00,327: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\test.pkl of shape:(7550, 2502)
[2022-06-26 20:29:00,629: INFO: stage_02_featurization]: >>>>> stage Featurization completed!<<<<<

[2022-06-26 20:30:54,655: INFO: stage_02_featurization]: 
********************
[2022-06-26 20:30:54,655: INFO: stage_02_featurization]: >>>>> stage Featurization started <<<<<
[2022-06-26 20:30:54,657: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-26 20:30:54,659: INFO: common]: yaml file: params.yaml loaded successfully
[2022-06-26 20:30:54,660: INFO: common]: created directory at: artifacts\features
[2022-06-26 20:30:54,903: INFO: common]: The input data frame artifacts\prepared\train.tsv of size (17450, 3) is read.
[2022-06-26 20:31:10,200: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\train.pkl of shape:(17450, 2502)
[2022-06-26 20:31:10,331: INFO: common]: The input data frame artifacts\prepared\test.tsv of size (7550, 3) is read.
[2022-06-26 20:31:13,297: INFO: data_mgmt]: The output matrix saved at artifacts\prepared\test.pkl of shape:(7550, 2502)
[2022-06-26 20:31:13,685: INFO: stage_02_featurization]: >>>>> stage Featurization completed!<<<<<

[2022-06-27 21:30:10,060: INFO: stage_03_training]: 
********************
[2022-06-27 21:30:10,062: INFO: stage_03_training]: >>>>> stage TRAINING_STAGE started <<<<<
[2022-06-27 21:30:10,066: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-06-27 21:30:10,067: ERROR: stage_03_training]: while scanning a simple key
  in "params.yaml", line 15, column 3
could not find expected ':'
  in "params.yaml", line 16, column 1
Traceback (most recent call last):
  File "src/stage_03_training.py", line 83, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_03_training.py", line 27, in main
    params = read_yaml(params_path)
  File "c:\users\lmohan2\dvc-project-docs-template\lm_dvc_nlp_project-with-docs\src\utils\common.py", line 11, in read_yaml
    content = yaml.safe_load(yaml_file)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\__init__.py", line 81, in load
    return loader.get_single_data()
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\parser.py", line 98, in check_event
    self.current_event = self.state()
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\scanner.py", line 115, in check_token
    while self.need_more_tokens():
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\scanner.py", line 152, in need_more_tokens
    self.stale_possible_simple_keys()
  File "C:\Users\lmohan2\Anaconda3\envs\env\lib\site-packages\yaml\scanner.py", line 292, in stale_possible_simple_keys
    "could not find expected ':'", self.get_mark())
yaml.scanner.ScannerError: while scanning a simple key
  in "params.yaml", line 15, column 3
could not find expected ':'
  in "params.yaml", line 16, column 1
